rfcs/text/0000-simple-updates.md

```diff
@@ -0,0 +1,161 @@
+* The wranglers can see what documents in a submission have changed via an ingest UI view. 
+
+
+### Unresolved Questions
+
+This RFC does not represent a complete solution to the Addition and Update functionality that is needed by the DCP. There are a number of unresolved questions and missing functionality that will be left for future RFCs to address.
+
+*   How is the provenance of data and metadata made clear to consumers?
+*   How are data retracted without creating meaningless 404 errors for consumers who had used it? [RFC4: Deletion of Data in the DCP]([https://github.com/HumanCellAtlas/dcp-community/blob/master/rfcs/text/0004-dss-deletion-process.md](https://github.com/HumanCellAtlas/dcp-community/blob/master/rfcs/text/0004-dss-deletion-process.md)) considers some of this challenge.
+*   How do we update files as opposed to metadata?
+*   How do we make the update process through Ingest more straightforward?
+*   What logic is needed to make decisions about rerunning data processing pipelines? Where should that logic live? Can that decision be automated?
+*   How do we carry out updates which change our representation of the experimental design and change the number or types of entities which are available as part of a submission?
+*   What is the interaction between metadata schema changes and updates?
+
+### Drawbacks and Limitations 
+
+There are many types of updates which are not supported by this solution including:
+
+1. Updating to the experiment design representation, ie adding or removing protocols and biomaterials or identification of incorrect relationships between existing biomaterials, processes and files.
+2. Updates which require additional or updated experimental data files
+3. Updates which require an analysis pipeline to be run again, either a new pipeline or the same pipeline with different parameters
+4. Migrating metadata to a new schema where this is necessary before a metadata update can occur. This is being considered independently of updates.
+5. Removing experimental data files from a submission.
+6. Redacting metadata (in the sense of completely purging it/hard deleting it from the system)
+7. Redacting data (in the sense of completely purging it/hard deleting it from the system)
+8. Updating metadata on unsubmitted datasets.
+
+Implementing all of these features will take the DCP time and we feel that the proposed design represents a step forward in the update and additional process and is valuable even if not complete. One important issue this design allows us to address which the old update process does not is it will give us an auditable trail of edits to bundles which can be displayed to users, improving their experience of and trust in the platform.
```

Comment:
*Editorial*: Implementing all of these features will take ~the DCP~ time ~and~, but ~we feel that~ the proposed design represents a step forward in the update and addition~al~ process ~and~, which is valuable even if ~not~ incomplete. ~One important issue this design allows us to address which the old update process does not is it will give us an auditable trail of edits to bundles which can be displayed to users~, Unlike the current update process, this design offers an auditable trail of edits to bundles which can be presented to users. improving their experience of and trust in the platform.

Reply:
Thanks for the suggestion, I hope I transferred it faithfully